<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AUST WiFi 双用户配置</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f5f5f5;
}

QLabel {
    color: #333;
    font-size: 12px;
    font-weight: bold;
}

QLineEdit {
    border: 1px solid #ccc;
    border-radius: 4px;
    padding: 10px;
    font-size: 12px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus {
    border-color: #0078d4;
}

QComboBox {
    border: 1px solid #ccc;
    border-radius: 4px;
    padding: 10px;
    font-size: 12px;
    background-color: white;
    min-height: 20px;
}

QComboBox:focus {
    border-color: #0078d4;
}

QCheckBox {
    font-size: 12px;
    color: #333;
    padding: 8px;
}

QGroupBox {
    font-weight: bold;
    border: 2px solid #ccc;
    border-radius: 5px;
    margin-top: 15px;
    margin-bottom: 10px;
    padding-top: 15px;
    padding-bottom: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 8px 0 8px;
    font-size: 13px;
}

QPushButton {
    background-color: #0078d4;
    border: none;
    border-radius: 4px;
    color: white;
    font-size: 13px;
    padding: 12px 20px;
    min-height: 30px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #106ebe;
}

QPushButton:pressed {
    background-color: #005a9e;
}

QPushButton#cancelButton {
    background-color: #d13438;
}

QPushButton#cancelButton:hover {
    background-color: #b52d31;
}

QPushButton#cancelButton:pressed {
    background-color: #9a272a;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QGroupBox" name="studentGroup">
     <property name="title">
      <string>👨‍🎓 学生配置</string>
     </property>
     <layout class="QFormLayout" name="studentLayout">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="studentUserLabel">
        <property name="text">
         <string>学号:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="studentUserEdit">
        <property name="placeholderText">
         <string>请输入完整学号（如：2023305123）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="studentPasswordLabel">
        <property name="text">
         <string>密码:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="studentPasswordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>请输入校园网密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="studentServerLabel">
        <property name="text">
         <string>运营商:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="studentServerCombo">
        <item>
         <property name="text">
          <string>电信</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>联通</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>移动</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="teacherGroup">
     <property name="title">
      <string>👨‍🏫 教师配置</string>
     </property>
     <layout class="QFormLayout" name="teacherLayout">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="teacherUserLabel">
        <property name="text">
         <string>工号:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="teacherUserEdit">
        <property name="placeholderText">
         <string>请输入工号（如：2021043）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="teacherPasswordLabel">
        <property name="text">
         <string>密码:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="teacherPasswordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>请输入校园网密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="teacherNote">
        <property name="text">
         <string>注：教师账号固定使用学校服务器，无需选择运营商</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #666; font-size: 11px; font-weight: normal; font-style: italic; padding-top: 10px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="autoStartCheck">
     <property name="text">
      <string>开机自动启动</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="authorLabel">
     <property name="text">
      <string>信息安全23-1 王智杰</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #888; font-size: 10px;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 